lib/connection.js

```diff
@@ -715,6 +715,15 @@ Connection.prototype.execute = function execute(sql, values, cb) {
   }
   this._resolveNamedPlaceholders(options);
 
+  // check for values containing undefined
+  if (options.values) {
+    options.values.forEach(function(val) {
+      if (val === undefined) {
+        throw new TypeError('Bind parameters must not contain undefined');
```

Comment:
Maybe expand a bit this message? Add something like " If you wanted to pass NULL value you have to use Javascript null explicitly"

Reply:
Updated to read 'Bind parameters must not contain undefined. To pass SQL NULL specify JS null'